lib/provider.js

```diff
@@ -31,27 +49,30 @@ module.exports = {
       return this.scanFeaturesDir(results)
     }
   },
-  scanStepDefinitionsDir: function(results) {
+  scanStepDefinitionsDir: function(results = []) {
     //TODO: first search step definitions for your file
-    for (var step_def_file of fs.readdirSync(`${this.rootDirectory()}${this.featuresDirectory()}/step_definitions`)) {
+    const searchPath = `${this.rootDirectory()}${this.featuresDirectory()}/step_definitions`;
+    const allStepPaths = this.searchForPattern(searchPath, /.*/);
+    allStepPaths.forEach( step_def_file => {
       let data = fs.readFileSync(`${this.rootDirectory()}${this.featuresDirectory()}/step_definitions/${step_def_file}`, 'utf8');
```

Comment:
Ahh, good, call. Sorry for the slow response time, works been crazy this week. I'll get this fixed up and finalized. By the way, do you have an example steps file that I could stick in the test directory? I know you said you use the Ruby implementation, would be nice to have a ruby step file to test with.